allauth/socialaccount/providers/base.py

```diff
@@ -24,6 +24,17 @@ class AuthError(object):
 
 
 class Provider(object):
+    alt_text = None
+    display_name = None
+    name = None
+
+    def __init__(self):
+        super(Provider, self).__init__()
```

Comment:
Since `Provider` is the lowest level Provider class, this `super().__init__()` call is not necessary.